### ISSUE TITLE
Output time precision (again) 

### ISSUE THREAD

**Issue by maximilian-gelbrecht** [SpeedyWeather/SpeedyWeather.jl#266]
We already had this in #180 and #186, but now after working with it again, I find it quite limiting to have the time output bound to full hours. Right now I am trying to line up two different models with different natural time units, so that it is a bit inconvenient to keep Speedy at a full-hour timestep for me. 

I would like the output time to be in Float64. We did discuss this before, and I don't really see a reason why Float64 and seconds since shouldn't work.

**Comment by milankl** [comment]
Wait so you want output to be possible in fractions of seconds? The time step in the dynamical core uses only full seconds, like you won't even get a time step that hits half a second so that you could output it

**Comment by milankl** [comment]
See also #237

**Comment by maximilian-gelbrecht** [comment]
Ah okay, I didn't know that the core is also in full seconds. Then, I retract the issue. I have one model where everything is scaled with 2\pi so I get a lot of non-integer values for everything, so it makes it a bit awkward to set up the speedy output and the output of my model to align with each other nicely.

**Comment by milankl** [comment]
I mean it's not impossible to change that, but I calculate what the time step in seconds and Float64 would need to be at a given resolution and round it down to the nearest integer in seconds and use that for the $2\Delta t$ in the leapfrog scheme (which actually is scaled by 1/radius) and in the `time += Dates.Second(Δt_sec)` of the main time loop which keeps track of the time of day and year the model is in (eventually important for shortwave radiation). But I'm not convinced doing this in Float64 would be better. Even at higher resolution like T255 as long as you match outputs within an hour it's usually good enough. Once we resolve radiation and clouds on the scale of minutes we may need to revise that, but for now?

**Comment by maximilian-gelbrecht** [comment]
Yeah, I am fine with that. At first I was thinking it is still in hours and Int64, which would be fairly limiting. But then I noticed it is actually seconds and Int64. It's quite an edge case that I have to deal with times scaled with Pi.
